notebooks/pandas/raw/indexing-selecting-assigning.ipynb

```diff
@@ -141,7 +169,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Uncomment the line below to see a solution\n",
+    "# Lines below will give you a hint or solution code\n",
+    "#_COMMENT_IF(PROD)_\n",
```

Comment:
You can use `#_COMMENT_IF(PROD)_` to comment a single line, in which case you write it twice here...

or you can use `#%%COMMENT_IF(PROD)%%` once at the top of the cell to comment each line. I personally prefer the cell-level macro because it saves one line of macros in that cell.

One of the risks of the single-notebook solution is that the testing code takes up a lot of space, so it can distract us from the view the user sees. If there's something we can do to save a line in the checking cells, I think that's worth doing. Just so our view is slightly closer to the user's view and we have a better sense of their experience next time we edit this notebook.

Reply:
sounds good, will do! i have not yet learned how to check that the notebooks render correctly.

Reply:
also wondering if a code cell like this (see pic) is good, or if you'd prefer i'd change it to not print out the incorrect dataframe? [to avoid distracting us from the view that the user sees]

<img width="899" alt="screen shot 2018-12-19 at 10 13 50 am" src="https://user-images.githubusercontent.com/10624937/50232533-cc4af700-0376-11e9-9a20-25be86e6d6a9.png">

Reply:
i found how to check the render in the documentation ... and need to make some edits. will add a new commit to fix the render soon
- `python render.py pandas` (run from notebooks directory)
- notebooks found in **pandas/default/rendered** folder

Reply:
> also wondering if a code cell like this (see pic) is good, or if you'd prefer i'd change it to not print out the incorrect dataframe? [to avoid distracting us from the view that the user sees]

In general we should exclude these unless there's an active reason to include it in this case. 80% of the time we work in this notebook in the future, we'll just be editing the wording of a question or something like that.  So a printout like this won't be relevant to us at that time (and since it doesn't get passed on to the user, having it doesn't do anything for them.)

Without having looked closely, it also seems like it might be showing something about how pandas works, and we have a high degree of trust in pandas.  

Again, there could be a one-off case I'm not thinking of where it's useful to be able to see this (e.g. if our checking code is doing something to the dataFrame that we want to visually check.)